x-pack/plugins/security_solution/public/common/components/links_to_docs/external_link.tsx

```diff
@@ -5,21 +5,34 @@
  */
 
 import React, { FC } from 'react';
-import { EuiLink } from '@elastic/eui';
+import { EuiLink, EuiToolTip } from '@elastic/eui';
 
 interface ExternalLinkProps {
   url: string;
-  text: string;
+  children?: React.ReactNode;
   ariaLabel?: string;
 }
 
 /**
- * A simplistic text link for opening external urls in a new browser tab.
+ * A link for opening external urls in a new browser tab.
  */
-export const ExternalLink: FC<ExternalLinkProps> = ({ url, text, ariaLabel }) => {
+export const ExternalLink: FC<ExternalLinkProps> = ({ url, children, ariaLabel }) => {
+  if (!children) {
+    return null;
+  }
+
   return (
-    <EuiLink href={url} aria-label={ariaLabel} external target="_blank" rel="noopener">
-      {text}
-    </EuiLink>
+    <EuiToolTip content={url} position="top" data-test-subj="externalLinkTooltip">
+      <EuiLink
+        href={url}
+        aria-label={ariaLabel}
+        external
+        target="_blank"
+        rel="noopener"
+        data-test-subj="externalLink"
+      >
+        {children}
+      </EuiLink>
+    </EuiToolTip>
```

Comment:
This guy needs unification with [`ExternalLink`](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/common/components/links/index.tsx#L85) component from `x-pack/plugins/security_solution/public/common/components/links/index.tsx`. I'll create a ticket for that and address separately targeting 7.12+, because this change can touch too many things in the UI.